tests/Unit/Filters/Audio/AudioMetadataTest.php

```diff
@@ -41,6 +41,7 @@ public function testAddArtwork()
         $filters = new AudioFilters($audio);
         $filters->addMetadata(array('genre' => 'Some Genre', 'artwork' => "/path/to/file.jpg"));
         $this->assertEquals(array(0 => "-i", 1 => "/path/to/file.jpg", 2 => "-map", 3 => "0", 4 => "-map", 5 => "1", 6 => "-metadata", 7 => "genre=Some Genre"), $capturedFilter->apply($audio, $format));
+        $this->assertEquals(array(0 => "-i", 1 => "/path/to/file.jpg", 2 => "-map", 3 => "0", 4 => "-map", 5 => "1", 6 => "-metadata", 7 => "genre=Some Genre"), $capturedFilter->apply($audio, $format));
```

Comment:
Not that's not useless, that's what I'm trying to explain. The *first* time the `apply` method was called the assertion passed. But if you wanted to `apply` again the same filter for a *second* time (e.g. to export to another format), then that line would make the test fail, because it would return a different array.